app/sqlitehelper.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+from sqlite3 import Connection, Error
+
+DEFAULT_DATABASE = "sqlitedatabase.db"
```

Comment:
Makes more sense to create a .env file that contains environment variables for the database connection. 

What you have now might be okay since it's only a local database though, and we do not necessarily need to hide database credentials.